dangerzone/podman/cli.py

```diff
@@ -22,7 +22,10 @@
 )
 def main(log_level: str) -> None:
     """Manage Dangerzone Podman machines."""
-    logging.basicConfig(level=getattr(logging, log_level.upper()), stream=sys.stderr)
```

Comment:
This removes `stream=sys.stderr`, as I believe it's the default behavior. Let me know if I'm missing anything here.